2-mandatory/1-weather-report.js

```diff
@@ -11,9 +11,26 @@
         - Hint: you can call the temperatureService function from your function
 */
 
-function getTemperatureReport(cities) {
-    // TODO
-}
+
+
+
+ function getTemperatureReport(cities) {
+
+let news = []
```

Comment:
This works fine - only issue is the code could be better spaced and indented.

Reply:
I will ~~~~thanks